rotkehlchen/globaldb/manual_price_oracle.py

```diff
@@ -0,0 +1,72 @@
+import logging
+from typing import Optional
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.constants import ZERO
+from rotkehlchen.errors import NoPriceForGivenTimestamp
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.history.typing import HistoricalPriceOracle
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.typing import Price, Timestamp
+from rotkehlchen.utils.misc import timestamp_to_date
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+class ManualPriceOracle:
+
+    def can_query_history(  # pylint: disable=no-self-use
+            self,
+            from_asset: Asset,  # pylint: disable=unused-argument
+            to_asset: Asset,  # pylint: disable=unused-argument
+            timestamp: Timestamp,  # pylint: disable=unused-argument
+            seconds: Optional[int] = None,  # pylint: disable=unused-argument
+    ) -> bool:
+        return True
+
+    @classmethod
+    def query_historical_price(
+        cls,
+        from_asset: Asset,
+        to_asset: Asset,
+        timestamp: Timestamp,
+    ) -> Price:
+        """
+        Query the historical price on `timestamp` for `from_asset` in `to_asset`.
+        So how much `to_asset` does 1 unit of `from_asset` cost.
+        Args:
+            from_asset: The ticker symbol of the asset for which we want to know
+                        the price.
+            to_asset: The ticker symbol of the asset against which we want to
+                      know the price.
+            timestamp: The timestamp at which to query the price
+        May raise:
+        - NoPriceForGivenTimestamp if we can't find a price for the asset in the given
+        timestamp from the user's manual price history.
+        """
+
+        price_entry = GlobalDBHandler().get_historical_price(
```

Comment:
```suggestion
        """
        price_entry = GlobalDBHandler().get_historical_price(
```

Unneeded whitespace.

Also the docstring is copy paste from the interface afaics so it's not needed I guess?